.gitignore

```diff
@@ -23,6 +23,11 @@ tools/modlog/converter.js
 *.tsbuildinfo
 **/sourceMaps
 
+/**/*.js
+!test/**/*.js
```

Comment:
should this also exclude tools/**/*.js? there's a bunch of JS files in there

Reply:
Aren't there some compiled files in there? I manually excluded the non-compiled ones.

Reply:
Did you? tools/simulate/index.js and tools/team-generation-benchmark.js seem to still be gitignored?